.rubocop.yml

```diff
@@ -1,3 +1,6 @@
+AllCops:
+  TargetSolidusVersion: 2.11
```

Comment:
This is to set the `TargetSolidusVersion`. 

User should set their version of Solidus here